Billmike/Chessecake-Factory
# Modified Template

### What does this PR do?
Modifies existing UI templates.
### Description of tasks to be completed?
- Modify existing user templates
### How should this be tested?
- After cloning repo to local machine, open up ```index.html``` to see modified changes
